src/binary-search-tree.js

```diff
@@ -5,6 +5,80 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+  insert(value) {
```

Comment:
Insert shouldn't be returning a boolean. If you want it to return something, it can return the newly-inserted BST node.